suministrospr/suministros/templates/suministros/suministro_form.html

```diff
@@ -46,10 +46,14 @@
       {{form.content}}
     </div>
 
-    <button class="bg-red-500 hover:bg-red-700 text-white font-bold py-2 px-4 rounded" type="submit">
-      Guardar y publicar
-    </button>
-    <a href="/" class="ml-2 text-red-500 hover:underline">Cancelar</a>
+    <div class="flex justify-center items-center">
+      <button class="w-full md:w-auto bg-red-500 hover:bg-red-700 text-white font-bold py-2 px-4 rounded" type="submit">
+        Guardar y publicar
+      </button>
+    </div>
+    <div class="flex justify-center items-center mt-4">
+      <a href="/" class="ml-2 text-red-500 hover:underline">Cancelar</a>
```

Comment:
@gcollazo que tal si hacemos algo así aquí? (redirect to detail view if editing, if creating al home)

```
href="{% if object %}{% url 'suministro-detail' object.slug %}{% else %}{% url 'suministro-list' %}{% endif %}"
```